report/renderer/swap-locale-feature.js

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
can this and `i18n-module` live in `viewer/`? It seems like the "feature" form is never going to be in the other reports

Reply:
I thought we'd be using this in LB?

Reply:
> I thought we'd be using this in LB?

maybe I'm wrong, but I thought from earlier conversations that the swapping part would be but not the UI/dropdown part?

Reply:
So I think we'll want to do the swapping in the main report view, but there's a link to the full report (but embedded in the LB site, so not a standalone report) where the UI would make sense to display.

It is unclear at this point, though.